server/app/models/grid_service.rb

```diff
@@ -147,7 +147,7 @@ def stack_exposed?
   end
 
   def deploy_pending?
-    self.grid_service_deploys.where(started_at: nil).count > 0
+    self.grid_service_deploys.where(finished_at: nil).count > 0
```

Comment:
If deploy has started but not finished is it really considered as pending or in-progress? Original idea behind `#deploy_pending?` was to check if service had deployment in a queue (but not yet started).